lib/simulation/openfisca/mapping/index.js

```diff
@@ -16,31 +16,26 @@ var isNotValidValue = function(value) {
 var getPeriods = function(dateDeValeur) {
     dateDeValeur = moment(dateDeValeur).freeze();
     return {
-        moisCourant: {
+        thisMonth: {
             key: dateDeValeur.format('YYYY-MM')
```

Comment:
The idea was to store them in a similar format than the other periods, which have a key, and sometimes also a start and an end. If you think it's too heavy, we can store only the values here and process the periods differently later.